doctr/models/detection/differentiable_binarization/pytorch.py

```diff
@@ -28,7 +28,8 @@
         'input_shape': (3, 1024, 1024),
         'mean': (0.798, 0.785, 0.772),
         'std': (0.264, 0.2749, 0.287),
-        'url': 'https://github.com/mindee/doctr/releases/download/v0.3.1/db_resnet50-ac60cadc.pt',
+        'url_straight_p': 'https://github.com/mindee/doctr/releases/download/v0.3.1/db_resnet50-ac60cadc.pt',
+        'url_rot_p': 'https://github.com/mindee/doctr/releases/download/v0.4.1/db_resnet50-1138863a.pt',
```

Comment:
If the checkpoint has superior perf, I suggest simply replacing the URL.

If it isn't, I would suggest to create a new entry in the config dict to keep things harmonized :+1: 

`db_resnet50_rotation`

Reply:
I think it makes the high level commands harder to use if we have 2 separate models, because users will have to specify the model in the predictor instead of just passing "assume_straight_pages=False", what do you think ?

Reply:
Ideally, the checkpoint should improve everything, and we should enable rotation by default and have a single checkpoint. If that's not the case, there should be one entry in the dict by checkpoint we make available

Otherwise, we will end up with various default dict format across all models (I just unified everything across all tasks :sweat_smile: #763 )

Reply:
In that case I think we will start with 2 entries before benchmarking both, because I am not sure the rotated checkpoints is better than the previous one on straight documents which represent 95% of the stream